src/Auditable.php

```diff
@@ -144,8 +144,7 @@ protected function resolveAuditExclusions()
     public function getAuditExclude(): array
     {
         if ( 
-            Config::get('audit.exclude_merge', false) || 
-            ($this->auditExcludeMerge ?? false) 
+            $this->auditExcludeMerge ?? Config::get('audit.exclude_merge', false)
```

Comment:
>It seems that you decided not to follow the instructions

the `false` is not necessary, the default is `null` and the null on a `if` is already `false`
```php
if ($this->auditExcludeMerge ?? Config::get('audit.exclude_merge')) {
```

Reply:
Apologies, now I understand. Code updated.

Reply:
>Code updated. 

the 'false' is still there